src/test/java/seedu/address/storage/StorageManagerTest.java

```diff
@@ -26,7 +31,9 @@
     public void setUp() {
         JsonAddressBookStorage addressBookStorage = new JsonAddressBookStorage(getTempFilePath("ab"));
         JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(getTempFilePath("prefs"));
-        storageManager = new StorageManager(addressBookStorage, userPrefsStorage, new RoomOccupancyStorage());
+        RoomOccupancyStorage roomOccupancyStorage = new RoomOccupancyStorage(Paths.get("nr"),
```

Comment:
i think better naming of the paths would be good? it's a little bit hard to understand/remember what the constructor of RoomOccupancyStorage is supposed to take in

Reply:
I just followed the name of files given above.